projects/components/src/stepper/stepper.component.scss

```diff
@@ -47,20 +47,23 @@
   @include flex-layout(column);
   justify-content: space-between;
 
-  .action-buttons {
+  .footer {
     position: absolute;
     bottom: 0;
     width: 100%;
     background-color: white;
     border-top: 1px solid $gray-1;
     padding-top: 8px;
-    display: flex;
-    justify-content: space-between;
-    z-index: 20;
 
-    .back-next {
+    .action-buttons {
       display: flex;
-      gap: 16px;
+      justify-content: space-between;
+      z-index: 20;
```

Comment:
Any specific reason for number `20`?

Reply:
Not added as part of this PR.

It was added before so that the footer is not covered by the content.